app/src/main/res/color/recent_bookmark_item_text_color.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
There's nothing wrong with using `<selector>` for the different text color state between light and dark theme. I personally would be interested in getting feedback from everyone on whether or not we should be moving towards doing it this way or just adding the color variables in `colors.xml` and `values-night/colors.xml` (https://searchfox.org/mozilla-mobile/search?q=home_show_all_button_text&path= for example).